conans/client/graph/sbom.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from conan.internal.cache.home_paths import HomePaths
+
+def migrate_sbom_file(cache_folder):
+    from conans.client.migrations import update_file
+    with open("spdx.py", "r") as file:
```

Comment:
What ``spdx.py`` is this opening? This is not very clear, please note that other plugins have implementation as strings, not as .py files.